includes/html/api_functions.inc.php

```diff
@@ -534,9 +534,9 @@ function maintenance_device(Illuminate\Http\Request $request)
     $alert_schedule = new \App\Models\AlertSchedule([
         'title' => $title,
         'notes' => $notes,
-        'behavior' => $behavior,
         'recurring' => 0,
     ]);
+    $alert_schedule->behavior = $behavior;
```

Comment:
Don't this, revert and add behavior to the fillable fields in the AlertSchedule model.